floodviz/views.py

```diff
@@ -18,8 +17,32 @@ def home():
 
     j = hydrograph_utils.req_hydrodata(sites, start_date, end_date, url_top)
     all_series_data = hydrograph_utils.parse_hydrodata(j)
-    
-    with open('floodviz/static/data/hydrograph_data.json', 'w') as fout: 
+
+    with open('floodviz/static/data/hydrograph_data.json', 'w') as fout:
         json.dump(all_series_data, fout, indent=1)
 
     return render_template('index.html')
+
+
+@app.route('/map/')
```

Comment:
Eventually this should go on the home page but we can take care of that in a subsequent pull request once we determine which way we want to go with the map stuff.